Composer/plugins/azurePublish/src/index.ts

```diff
@@ -258,7 +262,9 @@ class AzurePublisher {
       };
 
       // append provision resource into file
-      const resourcePath = path.resolve(this.getProjectFolder(resourcekey), 'appsettings.deployment.json');
+      // TODO: here is where we configure the template for the runtime, and should be parameterized when we 
```

Comment:
yes exactly, we'll need to hoist this "appsettings" stuff out

Reply:
https://github.com/microsoft/BotFramework-Composer/issues/2852 In r10 we should get all these harcodings out